From b0e3228924a25f914c674b471606c4c40c42087f Mon Sep 17 00:00:00 2001
From: Shuo Liu <shuo.a.liu@intel.com>
Date: Fri, 10 Aug 2018 16:33:08 +0800
Subject: [PATCH 2129/2328] vhm: change trace_printk of vhm_dev_ioctl to
 pr_debug

trace_printk will write into the ring buffer by default. Actually, we
don't need this trace entry as there are many vhm ioctls happen at
runtime which will flood the ring buffer. So change it to dynamic printk.

Signed-off-by: Shuo Liu <shuo.a.liu@intel.com>
Acked-by: Anthony Xu <anthony.xu@intel.com>
---
 drivers/char/vhm/vhm_dev.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/char/vhm/vhm_dev.c b/drivers/char/vhm/vhm_dev.c
index d367a6d..e80d1ce 100644
--- a/drivers/char/vhm/vhm_dev.c
+++ b/drivers/char/vhm/vhm_dev.c
@@ -158,7 +158,7 @@ static long vhm_dev_ioctl(struct file *filep,
 	long ret = 0;
 	struct vhm_vm *vm;
 
-	trace_printk("[%s] ioctl_num=0x%x\n", __func__, ioctl_num);
+	pr_debug("[%s] ioctl_num=0x%x\n", __func__, ioctl_num);
 
 	if (ioctl_num == IC_GET_API_VERSION) {
 		struct api_version api_version;
-- 
2.7.4

